cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -236,7 +244,7 @@ export class Assessment extends Application {
             if (applicationOpen) {
                 this.clickAssessButton();
                 cy.wait(SEC);
-                this.take_questionnaire();
+                this.take_questionnaire(questionnaireName);
```

Comment:
questionnaireName should have a default value (LegacyQuestionnaire)  else this line will throw error when questionnaireName is empty.